src/ImageSharp/Formats/Tiff/TiffEncoder.cs

```diff
@@ -17,6 +17,9 @@ namespace SixLabors.ImageSharp.Formats.Experimental.Tiff
     /// </summary>
     public class TiffEncoder : IImageEncoder, ITiffEncoderOptions
     {
+        /// <inheritdoc/>
+        public ByteOrder ByteOrder { get; } = TiffEncoderCore.ByteOrder;
```

Comment:
Is there any point exposing this in the encoder if we cannot change it?

Reply:
This is for information for user, according to the specification, tiff supports both byteOrder, so that he knows which byteOrder the file will be saved. I can make as internal?

Reply:
i think i will remove this, i agree that its not much of use here.